content/developer/Theme/sass_compiler.adoc

```diff
@@ -0,0 +1,400 @@
+---
+title: "Sass"
+weight: 4
+pre: "<b> </b>"
+---
+
+
+*From SuiteCRM version 7.8, the SuiteP theme is written in Sass. Which enables developers and system admins to easily configure the SuiteP theme; thanks to the variables.scss file. To ensure that your changes are not overidden by the compile/build process; In order to change the style of the SuiteP theme you MUST edit the sass files instead of the css files.*
+
+
+
+== About Sass
+
+'''
+Sass is an extension of CSS that adds power and elegance to the basic language. It allows you to use variables, nested rules, mixins, inline imports, and more, all with a fully CSS-compatible syntax. Sass helps keep large stylesheets well-organized, and get small stylesheets up and running quickly, particularly with the help of the Compass style library. +
+
+Please see the https://sass-lang.com/documentation[Sass Documentation] for more information about Sass. +
+ +
+ +
+
+=== About Sass Compliers and Git
+'''
+There are many Sass compilers available. So you can pick the one you prefer. However, you should not run multiple compilers at the same time as this can cause problems. Each compiler will generate the style.css file with different color and measurement formats. This can create unnecessary bloat in your git repository. So in order to reduce the file size of your git repository, before you commit style.css to git, you *should* use the Scss PHP Compiler. Please also only include the style.css in your last commit, this also helps to reduce the repository file size. The Scss PHP Compiler can be a little slow. You may wish to use an other compiler while you are working on the css. You may have to turn off the auto compilation / file watcher of your compiler before compiling with Sccs PHP. +
+ +
+ +
+
+=== Typical Workflow for Projects
+'''
+* Change a .scss file
+* Commit just the .scss file to git
+* Change a .scss file
+* Commit just the .scss file to git
+* Compile style.scss with the php compiler
+* Commit style.css +
+
+=== Typical Workflow for the Product
+'''
+* Change a .scss file
+* Commit just the .scss file to git
+* Change a .scss file
+* Commit just the .scss file to git
+* Make a pull request. Under the "How To Test" section" add "Rebuild sass". +
+
+
+_Don't commit style.css as we do not want any more bloat added to the product. Style.css will be built during the release process._ +
```

Comment:
This `more bloat` comment sounds a little too colloquial, and it sort of implies that we already have bloat in the project... ;-)

Please remove italic emphasis. If necessary use the `notice` blocks, but maybe it isn't necessary here.